docs/source/teams/installation.rst

```diff
@@ -320,26 +320,14 @@ here is an example configuration:
 
 .. code-block:: json
 
-    [
-        {
-            "AllowedHeaders": [
-                "*"
-            ],
-            "AllowedMethods": [
-                "GET",
-                "HEAD",
-            ],
-            "AllowedOrigins": [
-                "https://fiftyone-teams-deployment.yourcompany.com"
-            ],
-            "ExposeHeaders": [
-                "x-amz-server-side-encryption",
-                "x-amz-request-id",
-                "x-amz-id-2"
-            ],
-            "MaxAgeSeconds": 3000
-        }
-    ]
+[
```

Comment:
I believe code block content must be indented (like it previously was)

Reply:
then the closing brace should be indented too?

I was just going for consistency...